### PR TITLE
data:vidhimakani74

### PR DESCRIPTION
**Pull request by vidhimakani74**

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hi @vidhimakani74, Please replace the icons below with the respective icons for the links, else it will look something like this! 😄

image

File: data/vidhimakani74.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vidhimakani74",
+      "icon": "FaInstagram"
+    },
+     {
+      "name": "CodeChef",
+      "url": "https://www.codechef.com/users/vidhimakani007",
+      "icon": "FaInstagram"
```
[the remark this answers is not in the document]
Comment (vidhimakani74) [reply data/vidhimakani74.json:27]: i forgot to change icons, thanks for suggestions

File: data/vidhimakani74.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Vidhi Makani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Newbie in tech | Learning Web-Development | Learning from Communities & Opensource ",
+  
+  "tags": ["Open Source", "Web Development", "Fronted" ],
```
Comment (Panquesito7) [inline data/vidhimakani74.json:7]: Non-blocking: minor spelling error here.
```suggestion
  "tags": ["Open Source", "Web Development", "Frontend" ],
```

**Review by Panquesito7 — DISMISSED** [review]
Overall, it looks good. Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your contribution! 🎉

<p align="center"></p>

**Pull request merged**